src/codeflare_sdk/utils/generate_yaml.py

```diff
@@ -545,7 +545,7 @@ def write_user_appwrapper(user_yaml, output_file_name):
     print(f"Written to: {output_file_name}")
 
 
-def enable_openshift_oauth(user_yaml, cluster_name, namespace):
+def enable_openshift_oauth(user_yaml, cluster_name, namespace, route_item):
```

Comment:
We can remove `route_item` here it is basic dashboard route which is deleted in favour of the oauth dashboard route

Reply:
Thanks, have removed.